app/parser/v0_0_1/schema_parser.py

```diff
@@ -183,6 +183,8 @@ def _parse_block(self, schema, questionnaire):
         try:
             block.id = ParserUtils.get_required_string(schema, "id")
             block.title = ParserUtils.get_optional_string(schema, "title")
+            block.description = ParserUtils.get_optional_string(schema, "description")
```

Comment:
We should also define the optional as it will potentially be a tool for building schemas